docs/UserGuide.md

```diff
@@ -17,46 +17,51 @@ and efficient Graphical User Interface interaction.
 
 1. Copy the file to the folder you want to use as the _home folder_ for your CLI-nic system.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds.
+Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will display instructions on various commands. In addition, you can also type **`help COMMAND`** to understand the command format and sample commands associated with `COMMAND`.<br>
-   Some example commands you can try:
+1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will
+display instructions for the various commands. Typing **`help delete`** and pressing Enter will display in-depth
+instructions for the **`delete`** command.<br>Here are some sample commands to try:
 
    * **`add`** `ct/w n/warehouseA p/00000000 addr/John street, block 123, #01-01 r/First warehouse` : Adds a
-    warehouse with the name warehouseA located at John street, block 123, #01-01. This warehouse is the
-    “First warehouse”.
+    warehouse named `warehouseA` located at `John street, block 123, #01-01` to CLI-nic. This warehouse is the
+    `First warehouse`.
 
-   * **`add`** `ct/s n/Philips Pharmaceutical p/00000000 e/philipsPharm@gmail.com r/largest contractor` : Adds a
-    supplier named Philips Pharmaceutical. His contact number is 00000000 and his email is philipsPharm@gmail.com.
-    This supplier is the “largest contractor”.
+   * **`add`** `ct/s n/Philips Pharmaceutical p/00000000 e/philipsPharm@gmail.com r/Largest contractor` : Adds a
+    supplier named `Philips Pharmaceutical` with the phone number `00000000` and email `philipsPharm@gmail.com` to
+    CLI-nic. This supplier is the `Largest contractor`.
     
-   * **`assignmacro`** `a/uwm cs/update ct/w n/main warehouse` : Assign a macro that pairs the alias "uwm" to the command string "update ct/w n/main warehouse".
+   * **`assignmacro`** `a/uwm cs/update ct/w n/main warehouse` : Assigns a macro that pairs the alias `uwm` to the
+   command string `update ct/w n/main warehouse`.
 
-   * **`clear`** : Deletes all contacts.
+   * **`clear`** : Deletes all suppliers and warehouses entries in CLI-nic.
 
-   * **`delete`** `ct/s i/12` : Removes supplier at index 12 from the list of suppliers.
+   * **`delete`** `ct/s i/12` : Deletes the supplier at index 12 from the list of suppliers.
 
-   * **`edit`** `edit ct/s i/1 n/Alice p/68574214` : Edits supplier at index 1 in supplier list to have name Alice and phone number 685742141.
+   * **`edit`** `edit ct/s i/1 n/Alice p/68574214` : Edits the name and phone number of the supplier at index 1 in
```

Comment:
There is an extra edit here that should be removed

Reply:
Great catch!! Thanks :)